GUI/src/cz/fidentis/gui/comparison_batch/BatchRegistrationConfiguration.java

```diff
@@ -860,13 +871,13 @@ private Boolean areModelsLoaded(ProjectTopComponent tc) {
         return false;
     }
 
-    private void jComboBox6ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jComboBox6ActionPerformed
-        if (jComboBox6.getSelectedIndex() == 0) {
+    private void registrationMethodComboBoxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_registrationMethodComboBoxActionPerformed
+        if (registrationMethodComboBox.getSelectedItem() == RegistrationMethod.PROCRUSTES) {
```

Comment:
toto je zas celkom malickost ale predsa - tu by tiez stalo za to ulozit si do premennej `registrationMethodComboBox.getSelectedItem()`